smarts/core/tests/test_trajectory_interpolation_provider.py

```diff
@@ -300,10 +300,12 @@ def test_trajectory_interpolation_provider_in_smarts(smarts, agent_spec, scenari
         if agent_obs.events.reached_goal:
             reached_goal = True
             break
-
+    
+    # pytype: disable=attribute-error
     curr_position = agent_obs.ego_vehicle_state.position
     curr_heading = agent_obs.ego_vehicle_state.heading
     curr_speed = agent_obs.ego_vehicle_state.speed
+    # pytype: enable=attribute-error
```

Comment:
This seems odd to me that we would get an `attribute-error` here, it means that our observation object is not clearly defined.

Reply:
It thinks that `agent_obs` is `None`